### ISSUE TITLE
App Layout: Filter text size

### ISSUE THREAD

**Issue by daniellekirkwood** [vector-im/element-android#6816]
### Your use case

"Favourites" text size is small - is this as desired?

### Are you willing to provide a PR?

No

**Comment by SimonBrandner** [comment]
Yeah, this inconsistency always steals away my attention :(

**Comment by amshakal** [comment]
Yeah it's a tricky one. We went with the standard material design recommendation for tabs when text size gets too large but I personally am not a big fan of this approach myself.

**Comment by SimonBrandner** [comment]
> Yeah it's a tricky one. We went with the standard material design recommendation for tabs when text size gets too large but I personally am not a big fan of this approach myself.

I can imagine that working but not if the tab layout is the prominent part of the app with the tab names not being user-created :/

**Comment by amshakal** [comment]
I think you're right about that. It is a prominent part of the app. There are a few ways to resolve this:
1. We can limit the text size for the tabs so it always fits the width of your phone but can't guarnatee that will work for all translations. 
2. Currently we're trying to see if we can do swipe interactions to to switch between tabs, if we implement that, I can imagine improving the tabs by letting it extend beyond the screen similar to how telegram treats folders.

**Comment by nadonomy** [comment]
@amshakal huge +1 to switching to horizontally scrollable tabs. Letting the content bleed would solve a lot for multiple screen sizes and localisation.

**Comment by amshakal** [comment]
Good news on that front - on iOS it is possible (thanks to @gileluard o/ ) and on androird @fedrunov is investigating if it is possible. We can add this as a quick enhancement once the team has some bandwidth.

**Comment by SimonBrandner** [comment]
Related: https://github.com/vector-im/element-android/issues/7155
